vulture/core.py

```diff
@@ -489,40 +489,35 @@ def generic_visit(self, node):
 
 
 def _parse_args():
-    def csv(option, _, value, parser):
-        setattr(parser.values, option.dest, value.split(','))
-    usage = """\
-usage: %prog [options] PATH [PATH ...]
-
-Paths may be Python files or directories. For each directory vulture
-analyzes all contained *.py files.
-"""
+    usage = "%(prog)s PATH [PATH ...] [options]"
```

Comment:
Please leave [options] before PATH.

Reply:
Discussed in https://github.com/jendrikseipp/vulture/pull/119#discussion_r192597306